hazelcast/src/test/java/com/hazelcast/journal/AbstractEventJournalBounceTest.java

```diff
@@ -34,7 +34,7 @@
 import static com.hazelcast.spi.properties.GroupProperty.GENERIC_OPERATION_THREAD_COUNT;
 import static com.hazelcast.spi.properties.GroupProperty.PARTITION_COUNT;
 import static com.hazelcast.spi.properties.GroupProperty.PARTITION_OPERATION_THREAD_COUNT;
-import static com.hazelcast.test.starter.Utils.rethrow;
+import static com.hazelcast.util.ExceptionUtil.rethrow;
```

Comment:
I guess this was not an intended usage and motivated the renames for me (less options with the same method name help with the auto-import feature of IDEA).